src/main/java/com/minecolonies/blocks/BlockHutBuilder.java

```diff
@@ -1,5 +1,9 @@
 package com.minecolonies.blocks;
 
+import net.minecraft.block.state.IBlockState;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.block.state.IBlockState'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)